masterlist.yaml

```diff
@@ -742,14 +827,21 @@ plugins:
     tag:
       - C.Location
   - name: 'Alternate Start - Live Another Life.esp'
-    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/272/']
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/272/'
+        name: 'Alternate Start - Live Another Life on Nexus Mods'
+      - link: 'https://afkmods.iguanadons.net/index.php?/files/file/270-alternate-start-live-another-life/'
+        name: 'Alternate Start - Live Another Life on Afk Mods'
+      - link: 'https://bethesda.net/en/mods/skyrim/mod-detail/2918710'
+        name: 'Alternate Start - Live Another Life on Bethesda.net'
     global_priority: 70
     after:
       - 'Open Cities Skyrim.esp'
       - 'My Home Is Your Home.esp'
     tag:
       - C.Location
   - name: '3rdEraWeapMoS-lvlList.esp'
+  # Weapons of the Third Era
```

Comment:
Looks like you missed this when turning comments into named links?

Reply:
I was 50/50 on whether to add that one, so I left it. Added now.